examples/ComplexExample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "RNExperimental",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.4.2",
+    "react-native": "0.40.0",
+    "react-native-vector-icons": "^4.0.0",
+    "react-navigation": "git+https://github.com/react-community/react-navigation.git",
```

Comment:
Hi @annelorraineuy thanks for the info. Based on your recommendation, i updated rn-cli.config in the react-navigation root directory to include the example name  in the examples constant and it seems to be building successfully now. 

```
const examples = [
  'NavigationPlayground',
  'HelloHybrid',
  'LinkingExample',
  'Chat',
  'ComplexExample'
];
```

I've submitted a new pull request here - #1624 if you're interested. Many thanks again.